docs/Alert.md

```diff
@@ -0,0 +1,19 @@
+# Alert
+
+Alerts are blocks which contain information like errors.
+
+## Error
+
+```html
+<div class="cui__alert error">
```

Comment:
Maybe it should even be cui__alert--error like how the Paragraph component does it?